CHANGELOG.md

```diff
@@ -6,6 +6,153 @@
 
 Releases are now named after famous [Cricket](https://en.wikipedia.org/wiki/Cricket) players.
 
+## [0.5] - [TBD] - 2017-11-??
+
+JupyterHub 0.8, HTTPS & scalability.
+
+### Upgrading from 0.4
+
+#### DB upgrade
+
+This release contains a big JupyterHub version bump (from 0.7.2 to 0.8). If
+you are using the default database provider (sqlite), then the required db upgrades
+will be performed automatically when you do a `helm upgrade`.
+
+If you are using MySQL / PostgreSQL as your backing database, you will need to
+perform a manual database upgrade with the following steps:
+
+1. Make a full backup of your database, just in case things go bad.
+2. Make sure that the database user used by JupyterHub to connect to your database
+   can perform schema migrations like adding new tables, altering tables, etc.
+3. In your config.yaml, add the following config:
+
+   ```yaml
+   hub:
+     db:
+       upgrade: true
+   ```
+4. Do a `helm upgrade`. This should perform the database upgrade needed.
+5. Remove the lines you added in step 3, and do another `helm upgrade`.
+
+It is highly recommended that you have a `staging` environment that matches
+your production environment & can be used as a testbed for this upgrade. Feel
+free to reach out to us on [gitter](https://gitter.im/jupyterhub/binder) before
+your upgrade if you have any questions!
+
+#### Ingress config incompatibilities
+
+We've made HTTPS much easier to set up, with automated certificates from
+[Let's Encrypt](https://letsencrypt.org/) integration. However, this means
+that some of the keys used to set up your own ingress has changed.
+
+If you were using config under `ingress` purely to get HTTPS, we recommend
+that you delete your entire config section under `ingress` & instead follow
+the new [docs](<- link -> ) on getting HTTPS set up. It's much easier & a lot
+less error prone than the method recommended on 0.4.
+
+If you were using config under `ingress` for other reasons, you may continue
+to do so. The keys under `ingress` have changed, and are now much more in line
+with how many other projects use `ingress` in the [official charts repo](http://github.com/kubernetes/charts/).
+
+#### Custom images
+
+If you are using a custom built image, make sure that the version of the
+JupyterHub package installed in it is now 0.8. It needs to be version 0.7.2 for
+it to work with v0.4 of the helm chart, and needs to be 0.8 for it to work with
+v0.5 of the helm chart.
+
+#### Admin config incompatibility
+
+If you had used the `admin` config section before, you now need to move it under
+`auth`. So if you had config like:
+
+```yaml
+admin:
+   access: true
+   users:
+      - yuvipanda
+```
+
+it should now be:
+
+```yaml
+auth:
+  admin:
+    access: true
+    users:
+        - yuvipanda
+```
+
+### New Features
+
+#### JupyterHub 0.8
+
+JupyterHub 0.8 is full of new features - see [CHANGELOG](https://github.com/jupyterhub/jupyterhub/blob/master/docs/source/changelog.md#080-2017-10-03)
```

Comment:
link to the rendered docs: https://jupyterhub.readthedocs.io/en/latest/changelog.html#

Reply:
@minrk there seems to be no way to link to a specific release on that page though